src/contact.ts

```diff
@@ -222,12 +222,17 @@ export class Contact implements Sayable {
   }
 
   /**
-   * find contact by `name`(NickName) or `remark`(RemarkName)
+   * find contact by `name` or `alias`
    */
-  public static async findAll(queryArg?: ContactQueryFilter): Promise<Contact[]> {
+  public static async findAll(queryArg?: ContactQueryFilter | {remark: string | RegExp}): Promise<Contact[]> {
```

Comment:
move `remark: string | RegExp` to `ContactQueryFilter` with a comment noticing _DEPRECATED_

then you could use `.remark` again